pyanaconda/timezone.py

```diff
@@ -24,11 +24,11 @@
 
 import datetime
 import time
-import zoneinfo
 from collections import OrderedDict
 from functools import cache
 
 import langtable
+import zoneinfo
```

Comment:
Is this expected change in this commit? Looks to me like a bug from the rebase.

Reply:
Yeah that snuck in accidentally - I ran `ruff --fix` at some point, though I don't know why it would do that move. Will fix that in the next iteration.